include/verilated_vpi.cpp

```diff
@@ -1978,13 +2004,21 @@ vpiHandle vpi_iterate(PLI_INT32 type, vpiHandle object) {
         return ((new VerilatedVpioVarIter{vop, true})->castVpiHandle());
     }
     case vpiModule: {
```

Comment:
case vpiModule is the child type we want to iterate over, not the parent type. So any scope should be valid for iteration